edx_exams/settings/local.py

```diff
@@ -134,3 +134,15 @@
 # Lastly, see if the developer has any local overrides.
 if os.path.isfile(join(dirname(abspath(__file__)), 'private.py')):
     from .private import *  # pylint: disable=import-error
+
+EVENT_BUS_PRODUCER = 'edx_event_bus_kafka.create_producer'
+EVENT_BUS_CONSUMER = 'edx_event_bus_kafka.KafkaEventConsumer'
+EVENT_BUS_KAFKA_SCHEMA_REGISTRY_URL = 'http://edx.devstack.schema-registry:8081'
```

Comment:
shouldn't these be localhost url's as opposed to devstack for this file?

Reply:
@MichaelRoytman

Reply:
Actually, I think we should remove these settings. The event bus doesn't work outside of devstack. Sorry about that. Do you want me to make a pull request against `MST-1789-downstream-triggers` to remove these settings?